src/components/explore/ClientVolumeOverTimeGraph.svelte

```diff
@@ -34,6 +35,10 @@
     left={totalClientsGraph.left}
     right={totalClientsGraph.right}>
     {title}
+    <a
+      style="font-weight: 300; font-size: 0.7em; cursor: pointer;"
```

Comment:
Would it be better to have this in a class?

Reply:
Makes sense. Class added.